website/routes.py

```diff
@@ -855,6 +855,57 @@ def make_url_map(app):
         ),
 
 
+        ### Discussions ###
+
+        Rule(
+            [
+                '/project/<pid>/discussions/',
+                '/project/<pid>/node/<nid>/discussions/',
+            ],
+            'get',
+            project_views.comment.view_comments_project,
+            OsfWebRenderer('project/discussions.mako'),
+        ),
+
+        Rule(
+            [
+                '/project/<pid>/discussions/overview/',
```

Comment:
Instead of defining this filters as new routes, perhaps it'd be better to use query parameters for filtering (which is what they are typically used for):

```
GET /project/<pid>/discussions/?page="overview"
```